Doc/c-api/gcsupport.rst

```diff
@@ -173,3 +173,43 @@ if the object is immutable.
    this method (don't just call :c:func:`Py_DECREF` on a reference).  The
    collector will call this method if it detects that this object is involved
    in a reference cycle.
+
+
+Controlling the Garbage Collector State
+---------------------------------------
+
+The C-API provides the following functions for controlling
+garbage collection runs.
+
```

Comment:
Once you writes "Performs" and once "Enable" (no S). I suggest to avoid the S, most documentations avoid it.